src/index.ts

```diff
@@ -199,6 +199,11 @@ function generateStorybookFiles (options) {
     fileName: path.join('storybook', 'nuxt-entry.js'),
     options
   })
+  this.addTemplate({
```

Comment:
Thank for the PR 👍 
Do you mind adding a comment above this line and link to upstream issue?

Reply:
Done!